depends/libmc_target.sh

```diff
@@ -0,0 +1,31 @@
+#! /bin/bash
```

Comment:
If this works as expected when building for multiple targets, then we could rename it to `rust_targets.sh` and use it for `libain.mk` as well.

Reply:
updated filename.

Reply:
And using it in `libain.mk`?

Reply:
and `libain.mk`